torch/_inductor/compile_fx.py

```diff
@@ -844,11 +844,19 @@ def _compile_fx_inner(
 
     log.debug("FX codegen and compilation took %.3fs", time.time() - start)
 
-    # This message is for printing overview information of inductor mm counts, shapes,etc after lowering
-    if log.isEnabledFor(logging.INFO):
+    # This message is for printing overview information of inductor mm counts, shapes, etc after lowering
+    if log.isEnabledFor(logging.INFO) and len(counters["aten_mm_info"].items()):
         mm_table_data = []
         for key, value in counters["aten_mm_info"].items():
-            name, m, n, k = key.split("_")
+            # example key 1: aten.mm_128_128_128
+            # example key 2: aten._scaled_mm.default_128_128_128
+            # below is a bit brittle, but should work
+            parts = key.split("_")
+            if len(parts) < 4:
+                log.info(f"Unable to parse key {key}")  # noqa: G004
```

Comment:
Just use %s logging instead of fstring to properly clear the error. Since it's already in an f statement though it's fine.